src/parser.cpp

```diff
@@ -293,15 +294,28 @@ namespace sdptransform
 				return str;
 			}
 
+			case 'u':
+			{
+				std::istringstream iss(str);
+				std::uint64_t ll;
+
+				iss >> std::noskipws >> ll;
+
+				if (iss.eof() && !iss.fail())
+					return ll;
+				else
+					return 0u;
+			}
+
 			case 'd':
 			{
 				std::istringstream iss(str);
-				long long ll;
+				std::int64_t ll;
 
 				iss >> std::noskipws >> ll;
 
 				if (iss.eof() && !iss.fail())
-					return std::stoll(str);
+					return ll;
 				else
```

Comment:
Why this change?

Reply:
While I do agree this change is unrelated to the fix this PR solves, there is no need to convert again the string which was already converted to integer by the stringstream.

Reply:
Oh true.